python/tk_multi_publish2/dialog.py

```diff
@@ -235,8 +246,31 @@ def __init__(self, parent=None):
         self._display_action_name = self._bundle.get_setting("display_action_name")
         self.ui.publish.setText(self._display_action_name)
 
+        # UI tweaks based on the 'manual_load_enabled' property
+        # disabling drag & drop and browsing for files or folders.
+        #
+        # hide the drop area ( dashed-line border + plus icon )
+        self.ui.large_drop_area.setVisible(self.manual_load_enabled)
```

Comment:
did you try just hiding the `large_drop_area_frame`? Since it is the parent to all the widgets on that page, it is probably sufficient.

Reply:
Yes, you are right. Done.